### DISCUSSION TITLE
"如何保证消息队列的高可用？"那一节有误且过时了

### DISCUSSION THREAD

**Discussion by FLZ101** [doocs/advanced-java#184] [Ideas]
文中 RabbitMQ 的 "普通集群模式" 在官方文档中被称为 "federation" 而非 "clustering"。最新版本的 RabbitMQ 支持真正的集群模式，文中并未提到。

**Comment by chenqimiao** [comment]
@feng-lei 感谢提醒哈，你方便为这篇文章做一下提升么？

**Comment by FLZ101** [comment]
@chenqimiao 很乐意为之，但还在学习 RabbitMQ 的过程中，不知道什么时候可以来更新。
